src/AzureClient/AzureClient.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Quantum.Client" Version="0.12.20081729-beta" />
+    <PackageReference Include="Microsoft.Azure.Quantum.Client" Version="0.12.20072811-beta" />
```

Comment:
Did you mean to downgrade these packages, or is this just a merge resolution issue? (here and throughout the PR)

Reply:
This is currently the latest version which has the new package turned on; hopefully build 0.12.2008.1919 should be done soon, though.